book/02-using-atom/sections/02-text.asc

```diff
@@ -325,6 +325,11 @@ Bam, just fill that bad boy out and you have yourself a snippet. As soon as you
 
 The snippets functionality is implemented in the https://github.com/atom/snippets[atom/snippets] package.
 
+For more examples, see the snippets in the following languare packages: 
```

Comment:
What do you think about putting the links in the same line/sentence? I think that'd be a bit easier to read. For example:

```
For more examples, see the snippets in the https://github.com/atom/language-html/blob/master/snippets/language-html.cson[language-html] and https://github.com/atom/language-javascript/blob/master/snippets/language-javascript.cson[language-javascript] packages. 
```